lsp-mode.el

```diff
@@ -744,7 +744,8 @@ are determined by the index of the element."
                          (const :tag "Struct" Struct)
                          (const :tag "Event" Event)
                          (const :tag "Operator" Operator)
-                         (const :tag "Type Parameter" TypeParameter)))
+                         (const :tag "Type Parameter" TypeParameter)
+                         (const :tag "Other" nil)))
```

Comment:
Perhaps you should move this `Other` to the first row, so it will be on the same order of `lsp/symbol-kind-lookup`